utils.py

```diff
@@ -0,0 +1,380 @@
+__author__ = 'Michail Xyntarakis'
```

Comment:
I would look thru these and eliminate what we don't actually use in wrangler (maybe all?)